external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -2383,7 +2380,10 @@ fn field_defs_and_types(
             name: *fname,
             loc: floc,
         });
-        let doc_string = extract_doc_string(files, file_id_to_lines, &floc, Some(datatype_loc));
+        let doc_string = all_comments
+            .get(&floc.file_hash())
+            .and_then(|m| m.get(&floc.start()))
+            .cloned();
```

Comment:
This pattern is utilized enough here that I think it would make sense to pull this into a helper function and use that instead. E.g.,

```rust
fn get_doc_string(all_comments: &CommentMap, loc: Loc) -> Option<String> {
   all_comments.get(loc.file_hash()).and_then(|m| m.get(&loc.start())).cloned()
}
```

Reply:
Done!